### ISSUE TITLE
ExportTools not exporting

### ISSUE THREAD

**Issue by scattrbrain1994** [ukdtom/ExportTools.bundle#142]
PMS version: 1.18.1.1973
Plex Webclient version: 4.40.1
ExportTools version: V2.0.0.12 

Thanks ukdtom for your work on this app. 

My issue:
I just installed ExportTools via UAS. When I launch it via settings, and try to export a library or playlist, upon clicking save nothing happens. This is true whether or not I specify an export directory. 

Here are my logs:
[PMS Plugin Logs.zip](https://github.com/ukdtom/ExportTools.bundle/files/5106102/PMS.Plugin.Logs.zip)

**Comment by ukdtom** [comment]
Do you really use MD0_Data?

From the log:

`2020-08-21 00:30:16,882 (7f8a497fa700) :  DEBUG (__init__:425) - Entering ValidateExportPath
2020-08-21 00:30:16,883 (7f8a497fa700) :  DEBUG (__init__:430) - My master set the Export path to: /share/MD0_DATA/Export
2020-08-21 00:30:16,883 (7f8a497fa700) :  CRITICAL (__init__:449) - Bad Export Path eith error: Wrong path specified as export path (most recent call last):
  File "/home/scattrbrain/Library/Application Support/Plex Media Server/Plug-ins/ExportTools.bundle/Contents/Code/__init__.py", line 446, in ValidateExportPath
    raise Exception("Wrong path specified as export path")
Exception: Wrong path specified as export path`

And does `/share/MD0_DATA/Export` exists, since if not, it'll fail

**Comment by scattrbrain1994** [comment]
Oh I don't know what MD0_Data is but I just used `/share/MD0_DATA/Export` as an export path because it was explicitly mentioned as an example output path in [this section of the wiki](https://github.com/ukdtom/ExportTools.bundle/wiki/Configuration#export-path). This was after getting frustrated from many attempts using other other output paths. For example I tried simply putting `Export` as my output directory. I made sure there was a folder named `Export` on my home directory, but I'm not sure if that's where ExportTools expects it to be. Maybe that's the problem.

**Comment by scattrbrain1994** [comment]
**Attempt 1**
After setting the output path to `/Export` and creating an `Export` folder in what I assume is the first path of my Plex library - `/mnt/mpathag/scattrbrain/Library/Export` and also in my home directory - `/mnt/mpathag/scattrbrain/Export`
```
2020-08-21 17:14:53,788 (7fe272ffd700) :  DEBUG (__init__:430) - My master set the Export path to: /Export
2020-08-21 17:14:53,788 (7fe272ffd700) :  CRITICAL (__init__:449) - Bad Export Path eith error: Wrong path specified as export path (most recent call last):
  File "/home/scattrbrain/Library/Application Support/Plex Media Server/Plug-ins/ExportTools.bundle/Contents/Code/__init__.py", line 446, in ValidateExportPath
    raise Exception("Wrong path specified as export path")
Exception: Wrong path specified as export path
```
**Attempt 2**
After setting the output path to `/share/Export` and creating an `Export` folder within the already existing share directory inside my Plex library - `/mnt/mpathag/scattrbrain/Library/Plex/plexmediaserver-1.19.5.3112-b23ab3896.x86_64/usr/share/Export`
```
2020-08-21 17:07:14,558 (7ff2fdffb700) :  DEBUG (__init__:430) - My master set the Export path to: /share/Export
2020-08-21 17:07:14,558 (7ff2fdffb700) :  CRITICAL (__init__:449) - Bad Export Path eith error: Wrong path specified as export path (most recent call last):
  File "/home/scattrbrain/Library/Application Support/Plex Media Server/Plug-ins/ExportTools.bundle/Contents/Code/__init__.py", line 446, in ValidateExportPath
    raise Exception("Wrong path specified as export path")
Exception: Wrong path specified as export path
```

Where am I supposed to create the custom output path?

**Comment by ukdtom** [comment]
Create a share on your QNAP, and name it like `ExportTools`

Then define the output path to be: `/share/ExportTools`

And please report back

**Comment by ukdtom** [comment]
Or define as `/share/Public` and after that, the output will end up in your public share of your QNAP, but note that all users can access that!

**Comment by scattrbrain1994** [comment]
I should probably say that I'm not the administrator of this QNAP. I just have a slot on it for use as a seedbox, and I don't have root priveleges. Does that mean I won't be able to follow your instruction? I'm not sure what creating a share means. I'm not very linux-saavy.

**Comment by ukdtom** [comment]
Hmmmm......

If so, you have access to where the media is located ?
Meaning do you have share access to where the medias are located?

**Comment by scattrbrain1994** [comment]
To answer your question, I do have access to the medias, if medias means the music and movies etc that I stream via Plex. I store them in `/mnt/mpathag/scattrbrain/files/downloads/music` and `/mnt/mpathag/scattrbrain/files/downloads/films`.

But anyway the issue is solved! I went into the IRC channel for my seedbox provider and one of the admins chatting there was able to help me since he uses ExportTools. He said to just use the full path `/home/scattrbrain/Library/Export` instead of a relative path `/Export`. That did the trick!

The problem is that I mistook the `Library` folder in my home directory, which has a bunch of Plex-related folders inside it, as the location of my Plex Library, and figured that this is where ExportTools exports by default, based on [this part of the wiki](https://github.com/ukdtom/ExportTools.bundle/wiki/Configuration#export-path):

> When this plugin is launched for the first time, it will default to put the exported file in the same directory as the first path of the library you are trying to export.

I also assumed that if I specified a custom, relative output path, ExportTools would export to `/mnt/mpathag/scattrbrain/Library/<relative path>`

But the seedbox admin explained to me that my assumptions were wrong, that this library folder is just for Plex configuration data, and that when the wiki talks about library paths it means these folders: 
![firefox_9QH6Rw8Gs3](https://user-images.githubusercontent.com/68086286/90928096-3f1a3a00-e3c4-11ea-807a-187d0a1ff28a.png)

![firefox_hSM4ytS47h](https://user-images.githubusercontent.com/68086286/90928560-1b0b2880-e3c5-11ea-9cfd-7820ea6167b4.png)

So by default ExportTools exports to `/mnt/mpathag/scattrbrain/files/uploads/music` if I export my Music Library, and to `/mnt/mpathag/scattrbrain/files/uploads/films` if I export my Movies Library.

And if I want to specify a custom path, it must start from `/home/scattrbrain/`

Silly mistake on my part. Thanks for your help and the app itself!
